src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/decisionissued/DecisionIssuedAboutToSubmitHandlerTest.java

```diff
@@ -145,13 +145,14 @@ public void givenGenerateNoticeIsYes_thenReturnTrue() {
 
     @Test
     @Parameters({"strikeOut, struckOut", "null, struckOut", ",struckOut"})
-    public void givenDecisionTypeIsStrikeOut_shouldSetDwpStateValue(@Nullable String decisionType,
+    public void givenDecisionTypeIsStrikeOut_shouldSetDwpStateValueAndInterlocReviewState(@Nullable String decisionType,
                                                                     @Nullable String expectedDwpState) {
         sscsCaseData.setDecisionType(decisionType);
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
         String currentDwpState = response.getData().getDwpState();
         String assertionMsg = "dwpState value (%s) is not as expected (%s)";
         assertEquals(String.format(assertionMsg, currentDwpState, expectedDwpState), expectedDwpState, currentDwpState);
+        assertNull(response.getData().getInterlocReviewState());
```

Comment:
interlocReviewState filed is null before running this test. This test would pass even before you added the fix. Please, set the interlocReviewState to some value before running the test to ensure it set to null as part of the fix. Thanks